### ISSUE TITLE
[false-positive] "This statement seems to be disconnected from its parent foreach."

### ISSUE THREAD

**Issue by excentrik** [kalessil/phpinspectionsea#880]
Quite related to #621 , using the progress bar in the Symfony framework results in the warning "This statement seems to be disconnected from its parent foreach."

(Dummy) Example:
```php
    $progressBar = new ProgressBar($output);
    $progressBar->start();
    
    foreach ($rows as $row) {
        $this->addRowToCSV($row);
        $progressBar->advance(); // Inspection warning here
    }
    $progressBar->finish();
```

**Comment by kalessil** [comment]
Thank you for reporting @excentrik. Unfortunately, I don't a solution for this case, so a suppression is recommended here.

**Comment by kalessil** [comment]
Or introducing a link to the loop scope:
```php
foreach ($rows as $index => $row) {
        $this->addRowToCSV($row);
        $progressBar->setProgress($index);
    }
```
